### ISSUE TITLE
When set sess_driver to memcached nothing works anymore

### ISSUE THREAD

**Issue by PenguinArts** [magefly/CodeIgniter-Aauth#208]
I had to install memcached server and php extension for my project. After I succeeded in installing them, I changed the config.php with this:

$config['sess_driver'] = 'memcached';
$config['sess_cookie_name'] = 'ci_session';
$config['sess_expiration'] = 7200;
$config['sess_save_path'] = 'localhost:11211';
$config['sess_match_ip'] = FALSE;
$config['sess_time_to_update'] = 300;
$config['sess_regenerate_destroy'] = FALSE;

Now, I had a function where store in a variable all user details.  
$this->aauth->get_user(false) - NULL

Can anybody help me with this issue ?

Thanks!

**Comment by REJack** [comment]
Which CI Version & Aauth Version you use?

**Comment by PenguinArts** [comment]
Aauth version:  * @version 2.5.12
CI version: 3.1.5

**Comment by REJack** [comment]
I've tested memcached with CI 3.1.6 & Aauth v2.5.12 in one of my active projects and it works after restarting my webserver 😄 , did you tested your memcached without CI?

**Comment by PenguinArts** [comment]
No, but I installed this library for memcached: https://github.com/tomschlick/memcached-library I assume this causes the problems with session.
